schemas/common/telemetry/shared/v1.schema.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$id": "https://mapcolonies.com/common/telemetry/shared/v1",
```

Comment:
Doesn't `common` already mean `shared`? Why add it? Maybe just make it `telemetry/v1` or `telemetry/base/v1`.

Reply:
its config related only to telemetry.
`telemetry/v1` is not possible because of directory structure.
`telemetry/base/v1` is possible and might be more understandable.

Reply:
So maybe change to `telemetry/base/v1`?

Reply:
changed